qtodotxt/lib/file.py

```diff
@@ -50,16 +51,19 @@ def __str__(self):
     __repr__ = __str__
 
     def load(self, filename):
-
+        lines = ""
         try:
             with open(filename, 'rt', encoding='utf-8') as fd:
                 lines = fd.readlines()
         except FileNotFoundError:
-            raise ErrorLoadingFile("Trying to load a non-existing file: '{}".format(filename))
+            ErrorLoadingFile("Trying to load a non-existing file: '{}".format(filename))
+            return False
         except IOError as ex:                # deprecated since Python 3.3, it would be OSError for =>3.3-support
-            raise ErrorLoadingFile(str(ex))
+            ErrorLoadingFile(str(ex))
+            return False
```

Comment:
#318 is fine. I am just saying that one except line in load() should be enough and that we should do a try catch in  openFileNyName which display exception information if something goes wrong, this is mouch more pythonic than returning True/False from a method